examples/panel/stock-market-chatbot/requirements.txt

```diff
@@ -0,0 +1,16 @@
+panel==1.3.8 
+yfinance==0.2.37 
+hvplot 
+pandas 
+openai==1.12.0
+imagekitio
+holoviews==1.18.3
+duckdb==0.10.0
+bokeh==3.3.4
+python-dotenv
```

Comment:
remove this

Reply:
Resolved https://github.com/ploomber/doc/pull/124/commits/5134a06dd9966b18ae5cc687803e19a4ddf4f0c9